csrankings-c.csv

```diff
@@ -168,6 +168,7 @@ Carmem S. Hara,UFPR,https://www.inf.ufpr.br/carmem,NrgcBX8AAAAJ
 Carmem Satie Hara,UFPR,https://www.inf.ufpr.br/carmem,NrgcBX8AAAAJ
 Carmine Gravino,University of Salerno,http://www.unisa.it/docenti/carminegravino/english/index,fGpVbKwAAAAJ
 Carmine Ventre,King's College London,https://www.kcl.ac.uk/people/carmine-ventre,kx8cKycAAAAJ
+Carmit Hazay,Bar-Ilan University,https://www.eng.biu.ac.il/hazay/,vYF6cS0AAAAJ

```

Comment:
Computer Engineering, not CS (as with many of the others in this PR, including math). Please explain how they meet inclusion criteria (can solely advise PhD students for a PhD _in Computer Science_).